core/io/http_session_manager.hxx

```diff
@@ -153,8 +153,12 @@ class http_session_manager
                     session->start();
                     session->on_stop([type, id = session->id(), self = this->shared_from_this()]() {
                         std::scoped_lock lock(self->sessions_mutex_);
-                        self->busy_sessions_[type].remove_if([&id](const auto& s) { return !s || s->id() == id; });
-                        self->idle_sessions_[type].remove_if([&id](const auto& s) { return !s || s->id() == id; });
+                        self->busy_sessions_[type].remove_if([&id](const auto& s) {
```

Comment:
It looks like this is a code-formatter difference between @avsej and @thejcfactor ?  Did anything behavioural change here?

Reply:
Should be resolved.  Although need to see why the formatting is different.